FZJDDFCH/FZJDDFCH-IOC-01App/Db/error_setter.substitutions

```diff
@@ -0,0 +1,34 @@
+file $(UTILITIES)/db/error_setter.template {
+    pattern {P, STREAM_PV, PV_NAME}
+    
+    {"\$(P)", "FREQ:REF", "FREQ:SP:RBV"}
+    {"\$(P)", "FREQ:REF", "FREQ"}
+    {"\$(P)", "FREQ:REF", "PHAS:SP:RBV"}
+    {"\$(P)", "FREQ:REF", "PHAS"}
+    {"\$(P)", "FREQ:REF", "PHAS:STATUS"}
```

Comment:
PV naming conventions say that `STAT` is standard, rather than `STATUS`

https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming

Please fix all subsequent

Reply:
`STATUS` replaced with `STAT` in all PV names.  Some alias definitions also removed as they became self-referential.